packages/devtools_app/lib/src/screens/debugger/debugger_model.dart

```diff
@@ -422,7 +422,7 @@ Future<void> buildVariablesTree(
   } else if (instanceRef != null && serviceManager.service != null) {
     try {
       final dynamic result =
-          await _getObjectWithRetry(instanceRef.id!, variable);
+          await serviceManager.service!.getObject(variable.ref!.isolateRef!.id!, instanceRef.id!);
```

Comment:
nit: pull `variable.ref!.isolateRef!.id!` out into a variable above like it was before, just to make this a little more readable